src/portfolio.js

```diff
@@ -18,16 +18,47 @@ const greeting = {
 	portfolio_repository: "https://github.com/ashutosh1919/masterPortfolio",
 };
 
-const socialMediaLinks = {
+const socialMediaLinks = [
 	/* Your Social Media Link */
-	github: "https://github.com/ashutosh1919",
-	linkedin: "https://www.linkedin.com/in/ashutosh-hathidara-88710b138/",
-	gmail: "ashutoshhathidara98@gmail.com",
-	gitlab: "https://gitlab.com/ashutoshhathidara98",
-	facebook: "https://www.facebook.com/laymanbrother.19/",
-	twitter: "https://twitter.com/ashutosh_1919",
-	instagram: "https://www.instagram.com/layman_brother/"
-};
+	// github: "https://github.com/ashutosh1919",
+	// linkedin: "https://www.linkedin.com/in/ashutosh-hathidara-88710b138/",
+	// gmail: "ashutoshhathidara98@gmail.com",
+	// gitlab: "https://gitlab.com/ashutoshhathidara98",
+	// facebook: "https://www.facebook.com/laymanbrother.19/",
+	// twitter: "https://twitter.com/ashutosh_1919",
+	// instagram: "https://www.instagram.com/layman_brother/"
+  
+	{
+	  name: "Github",
+	  link: "https://github.com/ashutosh1919",
+	  className: "github",
+	  iconName: "fa-github",
+	},
+	{
+	  name: "LinkedIn",
+	  link: "https://www.linkedin.com/in/ashutosh-hathidara-88710b138/",
+	  className: "linkedin",
+	  iconName: "fa-linkedin-in",
```

Comment:
Users will consider `iconName` key as any other key like id, but this is special and we need to mention here that it is FontAwesome Icon name so my suggestion is that we should change this key to `fontAwesomeIcon`

Reply:
Yes, @ashutosh1919 I agree. Will modifying the key names and the code accordingly